### DISCUSSION TITLE
如果要重启一个项目是采用v2.10.0版本好还是v3.0.0-alpha.1的版本好呐？

### DISCUSSION THREAD

**Discussion by susanliy** [vbenjs/vue-vben-admin#3000] [Q&A]
目前想重新启一个项目，从0到1；是采用v2.10.0版本好还是v3.0.0-alpha.1的版本好呐？
想用新的，又怕不稳定，大伙有什么好的意见嘛？

**Comment by likui628** [comment]
那肯定是用2.10.0
v3好多demo都没有呢，你也不知道会有什么坑
除非你有能力解决这些问题

**Reply by susanliy** [reply]
那如果是v2.8.0版本与v2.10.0版本的话，选那个版本更为适合呐？

**Reply by likui628** [reply]
那选2.10吧
